package.json

```diff
@@ -32,9 +32,11 @@
     "babel-preset-es2015": "^6.3.13",
     "happiness": "github:jedwatson/happiness"
   },
+  "peerDependencies": {
+	  "react": "*",
```

Comment:
use 2 spaces.  I know typically we use tabs...but since npm automatically formats with spaces the next install would change this.